### DISCUSSION TITLE
Hacktoberfest badges

### DISCUSSION THREAD

**Discussion by andreaangiolillo** [community/community#36765] [Other Feature Feedback, Questions, & Ideas] [answered]
Happy Hacktober 2022! 🎃 🎃 

I was wondering if there is a way to add the Hacktoberfest badges to the profile achievements. Thanks!

**Comment by jfullstackdev** [comment, marked as answer]
as for the standard badges, as far as I know, you can't,

but on your personal landing README, you can add that as many as you want, if it's a format that can be pasted on the markdown

**Comment by ossanna16** [comment]
Thank you for asking a good question @andreaangiolillo, I'm glad you found our community 🙂

**Comment by HydraPhyzer** [comment]
> Happy Hacktober 2022! 🎃 🎃
> 
> I was wondering if there is a way to add the Hacktoberfest badges to the profile achievements. Thanks!

Hey @andreaangiolillo, Yes You Can Add Hacktoberfest Badges to your Github Profile. Your Hacktoberfest Badges are Linked to Your Holopin Account. Go and Signup on Holopin, to get all Badges. 

When you visit Holopin Website, You can see the Link, that you can attach to your GitHub Special Repository (Having the same name as your Github Profile Username).
